examples/tests/valid/sdk_tests/counter/reset.w

```diff
@@ -0,0 +1,15 @@
+bring cloud;
+
+let counter = new cloud.Counter(initial: 0);
+
+new cloud.Function(inflight () => {
+  assert(counter.peek() == 0);
+  counter.inc();
+  assert(counter.peek() == 1);
+  counter.inc();
+  assert(counter.peek() == 2);
+  counter.inc(10);
+  assert(counter.peek() == 12);
+  counter.reset();
```

Comment:
I see that reset can take one argument, can we test that too? (I can add it as well 🙂 )

Reply:
Awesome, yes - I added that case, thank you